src/Parser/Mage/Fire/Modules/Features/Checklist.js

```diff
@@ -0,0 +1,178 @@
+import CombustionSpellUsage from './CombustionSpellUsage';
+import HeatingUp from './HeatingUp';
+import HotStreak from './HotStreak';
+import MirrorImage from '../../../Shared/Modules/Features/MirrorImage';
+import RuneOfPower from '../../../Shared/Modules/Features/RuneOfPower';
+
+class Checklist extends CoreChecklist {
+  static dependencies = {
+    castEfficiency: CastEfficiency,
+    combatants: Combatants,
+    alwaysBeCasting: AlwaysBeCasting,
+    cancelledCasts: CancelledCasts,
+
+    cinderstorm: Cinderstorm,
+    combustionCharges: CombustionCharges,
+    combustionFirestarter: CombustionFirestarter,
+    combustionSpellUsage: CombustionSpellUsage,
+    heatingUp: HeatingUp,
+    hotStreak: HotStreak,
+    mirrorImage: MirrorImage,
+    runeOfPower: RuneOfPower,
+
+    legendaryUpgradeChecker: LegendaryUpgradeChecker,
+    legendaryCountChecker: LegendaryCountChecker,
+    prePotion: PrePotion,
+    enchantChecker: EnchantChecker,
+  };
+
+  rules = [
+    new Rule({
+      name: 'Always be casting',
+      description: <Wrapper><em><b>Continuously chaining casts throughout an encounter is the single most important thing for achieving good DPS as a caster</b></em>. There shoule be no delay at all between your spell casts, it's better to start casting the wrong spell than to think for a few seconds and then cast the right spell. You should be able to handle a fight's mechanics with the minimum possible interruption to your casting. Some fights (like Argus) have unavoidable downtime due to phase transitions and the like, so in these cases 0% downtime will not be possible.</Wrapper>,
+      requirements: () => {
+        return [
+          new Requirement({
+            name: 'Downtime',
+            check: () => this.alwaysBeCasting.downtimeSuggestionThresholds,
+          }),
+          new Requirement({
+            name: 'Cancelled Casts',
+            check: () => this.cancelledCasts.cancelledCastSuggestionThresholds,
+          }),
+        ];
+      },
+    }),
+    new Rule({
+      name: 'Managing  your procs',
+      description: <Wrapper>Fire Mage is heavily dependent on correctly using your <SpellLink id={SPELLS.FIRE_BLAST.id}/> and <SpellLink id={SPELLS.PHOENIXS_FLAMES.id}/> guaranteed crit abilities to properly convert <SpellLink id={SPELLS.HEATING_UP.id}/> to <SpellLink id={SPELLS.HOT_STREAK.id}/>. Remember to only use your guaranteed crit spells to convert Heating Up to Hot Streak, and also remember to precede each instant cast <SpellLink id={SPELLS.PYROBLAST.id}/> with a hardcast of <SpellLink id={SPELLS.FIREBALL.id}/>, <SpellLink id={SPELLS.SCORCH.id}/>, or <SpellLink id={SPELLS.PYROBLAST.id}/> (If you have <ItemLink id={ITEMS.MARQUEE_BINDINGS_OF_THE_SUN_KING.id}/>) to increase the chances of getting a new Heating Up or Hot Streak</Wrapper>,
+      requirements: () => {
+        return [
+          new Requirement({
+            name: "Fire Blast used without Heating Up",
```

Comment:
This stat might be better as an 'affirmative' stat, like the inverse number and "Fire Blasts used with Heaing Up"